readthedocs/templates/builds/build_detail.html

```diff
@@ -53,6 +53,14 @@
           {{ build.builder }}
         </li>
       {% endif %}
+
+      <div data-bind="visible: finished()">
+        <li>
+          <a href="{% url "build-log" build.pk  %}">
+            {% trans "View raw" %}
```

Comment:
I think this is kind of a _advanced user_ feature and I'd avoid exposing it to all the users since they probably don't care about this and it just adds more confusing things/elements. I mean, this is not something that you want to check/see as a final user. The log in the build page should be enough for the user and this API endpoint will be used by a script of similar by an advanced user.

On the other hand, I think this should be described in the documentation page of APIv2

Reply:
I think this feature would be very useful for debugging failed builds, since here is more easy to read/copy the text. That's why this feature was suggested #2921. And also would be very handy for us while helping a user, since they just could given the raw log url, or the file on github as an attach or embedded.

But I'm agreed that this section could be a little large, what about removing some of this info already provided by the raw log?

Reply:
If I search correctly, the docs for apiv2 are auto generated https://readthedocs.org/api/v2/

Reply:
I would remove this for all the user and make that link available only if `is_staff` (so, core developers can easily access while debugging). As I said, if the user wants to take a look at the log, can use the log that it's already displayed in a friendly way in the same page.

In the issue it's mentioned "easy to grep", which I agree and I don't think that final user will take a look at the log very often to make it part of the UX (considering that it's already ower loaded in that section)